bin/i18n/redact_restore_utils.rb

```diff
@@ -7,14 +7,24 @@
 require_relative 'i18n_script_utils'
 
 class RedactRestoreUtils
+  # RedactRestoreUtils manages the
+  def self.backup_source_file(source_path)
+    return unless File.exist?(source_path)
+    return unless source_path[I18N_SOURCE_DIR]
+    backup_path = source_path.sub(I18N_SOURCE_DIR, I18N_ORIGINAL_DIR)
+    I18nScriptUtils.copy_file(source_path, backup_path)
+  end
```

Comment:
We always need to make a backup of the files before the redaction. I added the `backup_source_file` so all source files are backed up through the same method.

Reply:
I wonder if `I18N_BACKUP_FILE_PATH`/`I18N_BACKUP_DIR_PATH` is useful here. Seems like the code expects that to be the place where backups are found, but it isn't used here. It was used in the now removed call to `copy_file`. Curious why some parts of the original script use `I18N_ORIGINAL_DIR` and other parts use the constructed `I18N_BACKUP_DIR` (which, admittedly, is constructed with `I18N_ORIGINAL_DIR`... so this likely works anyway, but I like/expect the symmetry.)

I might have expected:

```
    return unless source_path[I18N_SOURCE_DIR_PATH]
    backup_path = source_path.sub(I18N_SOURCE_DIR_PATH, I18N_BACKUP_DIR_PATH)
```

So it lines up with the sync scripts that are checking against `I18N_BACKUP_FILE_PATH`. Thoughts?

Reply:
`I18N_ORIGINAL_DIR` is where backups are expected to be found (`i18n/locales/original`). We arbitrarily use the term "original" to refer to the not-redacted version of files.
Before redaction, we create a backup of the "source" files in the "original" directory.
`I18N_BACKUP_FILE_PATH/I18N_BACKUP_DIR_PATH` is used in the restoration process when we need to provide a path to the unredacted file. 
I see your point and I could remove this method from the PR as it is unnecessary.
I thought having a more descriptive method would improve code readability.